dialog/color.go

```diff
@@ -166,16 +167,30 @@ func newColorButtonBox(colors []color.Color, icon fyne.Resource, callback func(c
 	return container.NewGridWithColumns(8, objects...)
 }
 
-func newCheckeredBackground() *canvas.Raster {
-	return canvas.NewRasterWithPixels(func(x, y, _, _ int) color.Color {
-		const boxSize = 8
+const (
+	boxSize       = 8
+	numberOfRings = 12
+)
 
+func newCheckeredBackground(radial ...bool) *canvas.Raster {
```

Comment:
This looks confusing, I appreciate that it's to keep the old call consistent but as it's internal it can be changed and I think it would be cleaner as just `bool`.